3/3scale-istio-adapter/Dockerfiles/build.sh

```diff
@@ -0,0 +1,58 @@
+# ----------------------------------------------------------------------------
+# Package       : 3scale-istio-adapter 
+# Version       : code-cleanups
+# Source repo   : https://github.com/3scale/3scale-istio-adapter
+# Tested on     : RHEL_8.2
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Krishna Harsha Voora
+#
+# Disclaimer: This script has been tested in non-root mode on given platform using
+#             the mentioned version of the package. It may not work as expected 
+#             with newer versions of the package and/or distribution.
+#             In such case, please contact "Maintainer" of this script.
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+# Download/Setup Golang 1.11
+cd /usr/local/
+wget https://golang.org/dl/go1.11.13.linux-ppc64le.tar.gz
```

Comment:
Was there a reason for this specific version of go?  did you know there is a ubi8 image for go called ubi8/go-toolset that should have everything you need?  It might speed up building this image, unless you are locked to that specific version of go?

Reply:
Thanks for the info Gerrit, I did not know this.
> did you know there is a ubi8 image for go called ubi8/go-toolset that should have everything you need?

Based on 3scale_istio_adapter's [CI Documentation](https://github.com/3scale/3scale-istio-adapter/blob/code-cleanups/.circleci/config.yml#L8) I had to stick with Go1.11 
>unless you are locked to that specific version of go?